src/core/lib/iomgr/parse_address.h

```diff
@@ -52,6 +54,15 @@ bool grpc_parse_ipv4_hostport(const char* hostport, grpc_resolved_address* addr,
 bool grpc_parse_ipv6_hostport(const char* hostport, grpc_resolved_address* addr,
                               bool log_errors);
 
+/** Populate \a resolved_addr to be a unix socket at |path| */
+grpc_error* grpc_fill_unix_path(const absl::string_view path,
```

Comment:
If we're introducing new functions here, let's put them in the `grpc_core` namespace and give them C++-style names.  I suggest something like `UnixSockaddrPopulate()` and `UnixAbstractSockaddrPopulate()`.

Reply:
Done